src/xmipp/libraries/reconstruction/angular_continuous_assign2.cpp

```diff
@@ -508,7 +508,14 @@ void ProgAngularContinuousAssign2::processImage(const FileName &fnImg, const Fil
 					steps(10)=steps(12)=1.; 
 				else {
 					steps(10)=steps(11)=steps(12)=1.;
-					currentDefocusU = currentDefocusV = currentAngle = std::numeric_limits<double>::quiet_NaN(); // initialize default values
+					if (hasCTF)
+					{
+						currentDefocusU = old_defocusU;
+						currentDefocusV = old_defocusV;
+						currentAngle = old_defocusAngle;
+					}
+					else
+						currentDefocusU = currentDefocusV = currentAngle = std::numeric_limits<double>::quiet_NaN(); // initialize default values
```

Comment:
Probably it does not affect because it is protected by "hasCTF" ifs, but doesn't it sound safer to initialize to 0?